src/Load.php

```diff
@@ -19,7 +19,12 @@ public static function load() {
 
 		define( 'WP_REPAIRING', true ); // Will not try to install WordPress
 		define( 'WP_CONTENT_DIR', ABSPATH . 'wp-content' );
-
+		if ( ! defined( 'UPLOADS' ) ) {
+		( defined ( '\dbless_UPLOADS' ) ) ?
+			define( 'UPLOADS', WP_CONTENT_DIR . '/' . constant('\dbless_UPLOADS') )
```

Comment:
```suggestion
		( defined ( 'dbless_UPLOADS' ) ) ?
			define( 'UPLOADS', WP_CONTENT_DIR . '/' . constant('dbless_UPLOADS') )
```
In these functions it is always fully qualified.